sprocs/grading_jobs_insert_internal.sql

```diff
@@ -53,9 +54,16 @@ BEGIN
 
     new_correct = (new_score >= 1.0);
 
+    -- Only update the graded_at time if the submission is gradable
```

Comment:
Are these changes meant to fix existing behavior or are they somehow related to this PR? A little confused since this is the sproc that handles internal grading jobs.

Reply:
Sorry, this is for #2195 , I’ve lumped both in here because the changes are similar.

Reply:
Ahh, makes sense, just wanted to double-check.